build.py

```diff
@@ -113,7 +113,7 @@ def run(self):
     // Execute after Closure has loaded.
     if (!window.goog) {
       alert('Error: Closure not found.  Read this:\\n' +
-            'developers.google.com/blockly/hacking/closure');
+            'https://developers.google.com/blockly/guides/modify-blockly/web/closure');
```

Comment:
Intentionally leave the https:// off so that it is easier to type.  This is not a clickable link.